apps/sensenet/src/components/Icon.tsx

```diff
@@ -284,12 +335,9 @@ export const IconComponent: React.FunctionComponent<{
   const repo = useRepository()
 
   const options: IconOptions = { style: props.style, injector, repo }
-  const resolvers = [
-    ...(props.resolvers || []),
-    ...defaultContentResolvers,
-    ...defaultSchemaResolvers,
-    ...defaultNotificationResolvers,
-  ]
+  const resolvers = props.resolvers
```

Comment:
I like the original solution better. use the prop. resolver first and if it fails fall back to the defaults. Can you please revert this back and write a customResolver just for the File type?